CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt

```diff
@@ -60,17 +60,18 @@ Needle Cost: 3
 Duration: 	2 Actions
 Range:		30m
 Effect:		You launch off three harmasist spikes in quick succession using both
-hands. Each has an 18 % chance to miss. You may pick up to three targets to 
-launch at. Each spike that hits deals 52 damage. If target has no 
-shield left and is still alive, apply a poison payload of your choice.
+hands. Each has an 2 chance to miss. You may pick up to three targets to 
+launch at. Each spike that hits deals 52 piercing damage and has an armor piercing 
+level of 3. If the needles defeat the armor of the target and the target is still alive, 
+apply a poison payload of your choice.
 
-----Peircing Spike	-----
+----Piercing Spike	-----
 Adrenaline Cost:35
 Needle Cost: 1
 Duration: 	2 Actions
 Range:		45m
-Effect:		10 % chance to miss. Deal 42 damage.
-ignores shields and ballistic armor. Apply a poison payload of your choice.
+Effect:		2 chance to miss. Deal 42 piercing damage. AP level 6.
+if the  needle defeats the target's armor, apply a poison payload of your choice.
```

Comment:
this doesn't say 'If target has no shield left," is that an intentional difference?
And it should say "pierces the target's armor" or "damages the targets health" because 'defeating' can be ambiguous: do you mean pierce the armor or go all the way through the armor points pool so that you damage health?